robot_lib.py

```diff
@@ -151,6 +152,37 @@ def dryGripper():
     daq_lib.gui_message("Dry gripper failed! " + e_s)
     setPvDesc("warmupThreshold",saveThreshold)          
     
+def DewarRefill(hours):
+  global _dewarRefillThread
+  seconds = int((hours * 60 * 60))
+  _dewarRefillThread = Thread(target=_dewarRefillTask, args=(seconds,0))
+  _dewarRefillThread.start()
+
+def _dewarRefillTask(seconds, flag):
+  global dewarRefillStop
+  DewarAutoFillOff()
+  DewarHeaterOn()
+  dewarRefillStop = 0
+  start_time = time.time()
+  goal_time = time.time() + (seconds)
+  while not dewarRefillStop:
+    if time.time() < goal_time:
+      time_remaining = round(((goal_time - time.time()) / 60), 1)
+      logger.info(f"DewarRefillTask: Time remaining until auto fill on... {time_remaining} minutes")
+      time.sleep(60)
+    else:
+      logger.info("Dewar refill task running.")
+      DewarAutoFillOn()
+      DewarHeaterOn()
```

Comment:
is this supposed to be `DewarHeaterOff()`? also, should this be called if cancel is called?

Reply:
I asked that specifically when we tested it. It is supposed to be on for both.